src/wallet/rpcwallet.cpp

```diff
@@ -3235,6 +3236,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "removeprunedfunds",        &removeprunedfunds,        {"txid"} },
 
     { "generating",         "generate",                 &generate,                 {"nblocks","maxtries"} },
+    { "wallet",             "rescanblockchain",         &rescanblockchain,         {"startheight", "stopheight"} },
```

Comment:
I think this was more appropriately positioned after "removeprunedfunds" as before. Probably after "move" makes the most sense, since it seems to be alphabetised.